### PR TITLE
Document how takeover works

### PR DESCRIPTION
**Pull request by rahul-thakoor**
Change-type: patch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,4 +270,97 @@ or
...
+- change root filesystem via `pivot_root`
+- Spawn Stage2 worker process
+
+#### Stage2 Migration Worker
+- setup Stage2 logging to external device is configured
+- Kill running processes
+- Copy required files to RAMFS
+- unmount partitions
+- Flash balenaOS image to disk
+- Validate if image was written successfully
```
Comment (lmbarros) [inline README.md:363]: Not a real review comment, just curious: what happens if the validation fails?

**Review by lmbarros — COMMENTED** [review]
I added a few minor comments, but other than that I think we are good to go with this one!

File: README.md (hunk 1 above)
Comment (rahul-thakoor) [reply README.md:363]: we simply log that validatin was not successful 
https://github.com/balena-os/takeover/blob/master/src/stage2.rs#L1076

**Review by lmbarros — APPROVED** [review]
:rocket:

**Pull request merged**